src/js/jooa11y.js

```diff
@@ -1,3 +1,6 @@
+import tippy from 'tippy.js';
```

Comment:
Please remove this from here, it is wrong import and prevent use `jooa11y.js` as module in browser.
You can prepend it in build scrip, as I did in my PR.

Reply:
> it is wrong import and prevent use jooa11y.js as module in browser.

I'm not sure why you think this wrong tippy has clearly an ESM export: https://github.com/atomiks/tippyjs/blob/60aee16c934fc861569f84a37c6a1a7f42a33fa0/package.json#L6

Javascript doesn't care about node/browser as long as the dependencies exist in the context of the environment in the script is used. Also, this doesn't produce some errors so I'll take it as "it works as it was intended"

Reply:
Or try change it to `import tippy from './tippy.js';`

Reply:
> Javascript doesn't care about node/browser

It care much, for Browser it is not supported import, it works only for Node.

Check https://stackoverflow.com/a/66920868/1000711
And linked issue there:

> It wouldn't work in pure Javascript because so-called “bare” module specifiers are currently not supported in ECMAScript modules specification. Module specifiers must be full URLs, or relative URLs starting with /, ./, or ../.

Reply:
> It care much, for Browser it is not supported import, it works only for Node.

You're confusing direct import from the source with bundling using a bundler. We are not importing tippy in the bowser from the source. We import it in the scr of the script which will then be bundled by rollup (fixing the `Uncaught TypeError: Failed to resolve module specifier "@popperjs/core"`). Check the final script (umd or esm, there are no imports)

Reply:
Yes I know, but I want `jooa11y.js` to be a perfect module :wink: 
So someone can import it and use in Browser. And this import breaks whole thing.
It like 'tippy.esm.js' that even not possible to use without another compilation, because `'@popperjs/core'` :hankey:

Reply:
> So someone can import it and use in Browser. And this import breaks whole thing.

It is. Check it in your browser, also the script.js imports the `jooa11y.js` and works as expected. Everything that is needed is bundled, similar to the choices.js that bundles preact (there's no external dependency for preact in choices and similarly there's no external dependency for tippy or popper, everything that is needed is already encapsulated here)

Reply:
> And yes, I know we will have a bundle in dist, but I still want it to be workable even in source

Most of the ESM modules that have dependencies don't work out of the box in the browser unless the dependency is pointing to a URL. This is the reason that most scripts need a bundler with a node resolve plugin to fix those paths for the browser.

Reply:
well, okay, leave it as it is, 
yea, this tippy still useless as ESM module in browser, so it not really possible to do what I want :smile: